Library/Homebrew/test/test_options.rb

```diff
@@ -56,7 +56,7 @@ def test_equality
 
 class OptionsTests < Homebrew::TestCase
   def setup
-    @options = Options.new
+    @options = Options.create
```

Comment:
The idea is that `Options.new` will become a private implementation detail after this PR, it should not be called outside of `Options`.